src/react-components/src/__tests__/components/Template-test.tsx

```diff
@@ -0,0 +1,57 @@
+import { expect } from "chai";
+import { stub } from "sinon";
+import * as Enzyme from "enzyme";
+import * as React from "react";
+import Template from "../../components/Template/Template";
+import { LayoutTypes } from "../../components/Template/LayoutTypes";
+import Placeholder from "../../components/00-base/Placeholder";
```

Comment:
I think this isn't importing from the right place

Reply:
will fix